models/prices/prices_tokens.sql

```diff
@@ -600,7 +600,6 @@ VALUES
     ("hop-hop-protocol", "ethereum", "HOP", "0xc5102fe9359fd9a28f877a67e36b0f050d81a3cc", 18),
     ("x2y2-x2y2token", "ethereum", "X2Y2", "0x1e4ede388cbc9f4b5c79681b7f94d36a11abebc9", 18),
     ("tcr-tracer-dao", "ethereum", "TCR", "0x9c4a4204b79dd291d6b6571c5be8bbcd0622f050", 18),
-    ("wsteth-wrapped-liquid-staked-ether-20", "ethereum", "WSTETH", "0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0", 18),
```

Comment:
This was duplicated see main:

https://github.com/duneanalytics/spellbook/blob/0c5b439f2006c96f045c4c50feabcb82a3629209/models/prices/prices_tokens.sql#L588-L604


Added in both #1628 and #1639